test/interface.jl

```diff
@@ -564,17 +580,17 @@ end
     res1 = forward(sb1, [x, x, y, y])
     @test sb1(param([x, x, y, y])) isa TrackedArray
 
-    @test sb1([x, x, y, y]) == res1.rv
+    @test sb1([x, x, y, y]) ≈ res1.rv
     @test isapprox(logabsdetjac(sb1, [x, x, y, y]), 0, atol = 1e-6)
     @test isapprox(res1.logabsdetjac, 0, atol = 1e-6)
 
     sb2 = Stacked([b, b, inv(b), inv(b)])        # <= Array
     res2 = forward(sb2, [x, x, y, y])
     @test sb2(param([x, x, y, y])) isa TrackedArray
 
-    @test sb2([x, x, y, y]) == res2.rv
-    @test logabsdetjac(sb2, [x, x, y, y]) ≈ 0.0
-    @test res2.logabsdetjac ≈ 0.0
+    @test sb2([x, x, y, y]) ≈ res2.rv
+    @test isapprox(logabsdetjac(sb2, [x, x, y, y]), 0.0, atol = 1e-12)
```

Comment:
Yeah we should avoid checks of the form `x ≈ 0.0` since by default `atol = 0`, hence they will always fail if `x` is not exactly zero (there's a warning in the doc string). IIRC you could also write `@test x ≈ 0.0 atol=1e-12` if you think that's visually more appealing.

Reply:
> Yeah we should avoid checks of the form x ≈ 0.0 since by default atol = 0

Ah, didn't know that; thanks!

> IIRC you could also write @test x ≈ 0.0 atol=1e-12 if you think that's visually more appealing.

I do! Will change those:)